speedy-recovery-client/src/components/core/App.js

```diff
@@ -328,47 +354,36 @@ class App extends Component {
         const carePlans = carePlanResource.map(carePlan =>
           fhirMapCarePlan(carePlan, this.state.fhirVersion)
         );
-        console.log("CAREPLAN RESOURCE AFTER MAPPING:",carePlans);
-        this.setState({carePlans});
+        this.setState({ carePlans });
+        console.log("CAREPLAN RESOURCE AFTER MAPPING:", carePlans);
+        this.setState({ carePlans });
       })
       .catch(error => {
         console.error(error);
       });
   }
 
-  updateStatePatient(patientResource) {
```

Comment:
Since we dont need this method anymore you can also remove the code in App.js where you fetch the patient information for the selected patient when you login?